doc/model_persistence.rst

```diff
@@ -55,6 +55,19 @@ with::
    available `here
    <https://joblib.readthedocs.io/en/latest/persistence.html>`_.
 
+When a estimator is unpickled with a scikit-learn version that is inconsistent
+with the version the estimator was pickled with, a
+:class:`~sklearn.exceptions.InconsistentVersionWarning` is raised. This warning
+can be caught to obtain the original version the estimator was pickled with:
```

Comment:
```suggestion
can be caught to obtain the original version for which the estimator was pickled with:
```

Reply:
I'd either leave it as is or maybe "...original version with which the estimator...". Slight preference for "leave as is"

Reply:
I had trouble parsing "the original version the estimator". @betatim isn't it missing something there (not a native speaker so I am surely wrong ;))

Reply:
I can't explain it (the grammar) :-/

Reply:
:) If you find it OK then it should be right then ;) Resolving this post.